src/transformers/tokenization_utils.py

```diff
@@ -359,10 +367,21 @@ def __init__(self, **kwargs):
         self._added_tokens_decoder.update(kwargs.pop("added_tokens_decoder", {}))
         self._added_tokens_encoder: Dict[str, int] = {k.content: v for v, k in self._added_tokens_decoder.items()}
 
-        # 4 init the parent class
+        # 4. init the parent class
         super().__init__(**kwargs)
 
-        # 4. If some of the special tokens are not part of the vocab, we add them, at the end.
+        # 5. Set the behavior when encountering an OOV token id
+        # Note: this must be set before _add_tokens, because _add_tokens 
+        # relies on the behavior of the OOV token ids for some tokenizers.
+        self.oov_error = kwargs.pop("oov_error", "")
+        # TODO @ArthurZ in version 4.42
+        if self.oov_error == "":
+            logger.warning_once(
+                "The `oov_error` argument is set to the default `'replace'`. It will default to `'strict'` in `transformers` (v4.42) "
+            )
```

Comment:
```suggestion
            logger.warning_once(
                "The `oov_error` argument is set to the default `'replace'`. It will default to `'strict'` in `transformers v4.42."
            )
```